pkgs/development/python-modules/pyzstd/default.nix

```diff
@@ -25,8 +28,17 @@ buildPythonPackage rec {
         --replace-fail '"setuptools>=64,<74"' '"setuptools"'
   '';
 
+  dependencies = [
+    typing-extensions
+  ];
+
   nativeBuildInputs = [
```

Comment:
Remove this.

Reply:
`pythonRuntimeDepsCheckHook` fails without all required dependencies. I do not understand why you want me to remove this.

Reply:
So do you want me to put `typing-extensions` inside of `build-system`?

Reply:
No. Please read the docs about the difference between build-system and dependencies.

Reply:
I actually read it beforehand, that's why I was confused by your comment. You pointed out that `typing-extensions` is not a runtime dependency, which is true. According to [the docs](https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-function), `dependencies` is for packages that are listed in `install_requires` (https://github.com/Rogdham/pyzstd/blob/475f1c0ee10fd58cc28aa296fccc1aaf7e54fee9/setup.py#L207). I guess I'll revert my commit then?

Reply:
> You pointed out that `typing-extensions` is not a runtime dependency

I didn't and it is.
I only asked you to remove nativeBuildInputs.